code/world.dm

```diff
@@ -642,6 +642,7 @@ var/f_color_selector_handler/F_Color_Selector
 
 	lagcheck_enabled = 0
 	processScheduler.stop()
+	SEND_SIGNAL(preMapLoad, COMSIG_GLOBAL_REBOOT)
```

Comment:
Should this use the define GLOBAL_SIGNAL?
```suggestion
	SEND_SIGNAL(GLOBAL_SIGNAL, COMSIG_GLOBAL_REBOOT)
```

Reply:
Err, right, I need to fix that. Sadly it's not that easy due to preprocessor weirdness. I'll probably make a SEND_GLOBAL_SIGNAL macro instead.